TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,180 @@ A tile is composed of a header section immediately followed by a body section.
+in the Cesium implementation of 3D Tiles.
 
-## Batch Table
+## Feature Table
+
+Contains values for `i3dm` semantics used to create instanced models.
+
+### Semantics
+
+#### Instance Semantics
+
+These semantics map to an array of feature values that are used to create instances. The length of these arrays must be the same for all semantics and is equal to the number of instances.
+
+If a semantic has a dependency on another semantic, that semantic must be defined in order to be a valid tile.
+
+| Semantic | Data Type  | Description | Required |
+| --- | --- | --- | --- | --- |
+| `POSITION` | `float32[3]` | A 3-component array of numbers containing `x`, `y`, and `z` Cartesian coordinates for the position of the instance. | :white_check_mark: Yes, if `POSITION_QUANTIZED` is not defined|
+| `POSITION_QUANTIZED` | `uint16[3]` | A 3-component array of numbers containing `x`, `y`, and `z` in quantized Cartesian coordinates for the position of the instance. | :white_check_mark: Yes, if `POSITION` is not defined <br> :large_blue_diamond: Depends on `QUANTIZED_VOLUME_OFFSET` <br> :large_blue_diamond: Depends on `QUANTIZED_VOLUME_SCALE` |
+| `NORMAL_UP` | `float32[3]`| A unit vector defining the `up` direction for the orientation of the instance. | :red_circle: No <br> :large_blue_diamond: Depends on `NORMAL_RIGHT` |
+| `NORMAL_RIGHT` | `float32[3]` | A unit vector defining the `right` direction for the orientation of the instance. Must be orthogonal to `up`. | :red_circle: No <br> :large_blue_diamond: Depends on `NORMAL_UP` |
+| `NORMAL_UP_OCT32P` | `uint16[2]` | An oct-encoded unit vector with 32-bits of precision defining the `up` direction for the orientation of the instance. | :red_circle: No <br> :large_blue_diamond: Depends on `NORMAL_UP_OCT32P` |
+| `NORMAL_RIGHT_OCT32P` | `uint16[2]` | An oct-encoded unit vector with 32-bits of precision defining the `right` direction for the orientation of the instance. Must be orthogonal to `up`. | :red_circle: No <br> :large_blue_diamond: Depends on `NORMAL_RIGHT_OCT32P` |
+| `SCALE` | `float32` | A number defining a scale to apply to all axes of the instance. | :red_circle: No |
+| `SCALE_NON_UNIFORM` | `float32[3]` | A 3-component array of numbers defining the scale to apply to the `x`, `y`, and `z` axes of the instance. | :red_circle: No |
+| `BATCH_ID` | `unit16` | The `batchId` of the instance that can be used to retrieve metadata from the `Batch Table`. | :red_circle: No |
+
+#### Global Semantics
+
+These semantics define global properties for all instances.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `INSTANCES_LENGTH`| `uint32` | The number of instances to generate. The length of each array value for an instance semantic should be equal to this. | :white_check_mark: Yes |
+| `QUANTIZED_VOLUME_OFFSET` | `float32[3]` | A 3-component array of numbers defining the offset for the quantized volume. | :red_circle: No |
```

Comment:
This and below are required if `POSITION_QUANTIZED` is defined.  Overall, I'm not sure if the yes/depends/no column is clear.

Maybe it should be just:
* Yes
* Yes, if ...
* No

And drop the "depends on" since that becomes explicit in "Yes, if ..."